src/js/components/ContainerSettingsComponent.jsx

```diff
@@ -50,14 +67,18 @@ var ContainerSettingsComponent = React.createClass({
   },
 
   getPortMappingRow: function (i = 0) {
+    var props = this.props;
+    var model = props.model;
+    var errors = props.errors;
+
     return (
       <div key={`pm-${i}`} className="row duplicable-row">
         <div className="col-sm-3">
           <FormGroupComponent
```

Comment:
I know this is not part of this PR, but just another suggestion:
These guys could be made dynamically as well.